code4lib/2022.code4lib.org
# Iss1 - Improve Hero Image

This PR:

- Improves the responsiveness of the hero image
- Adds an optional webp alternative to the hero image for a further reduced size option, when the browser supports it
- Adds alt-text for the hero image
- Reduces the file size for the PNG version
- Adds license information and modifies logic in the template for missing attribution info
- Removes old images from last year's site

How to test:

- Verify there is no stretching of the hero image at different screen sizes and no whitespace around the image
- Check the image attributions page, ensuring license isn't displayed

Resolves #1, resolves #23
